# Week 1 Genome Assembly -- Feedback

1 + 0.5 + 0.5 + 1 + 0.75 + 1 + 1 + 0.9075 + 1 + 1 = 8.66 points out of 10 possible points

1. Question 1.1, 1.4 how many reads (0.5 pts each)

  * good --> +1

2. Question 1.2, 1.4 simulation script(s)

  * only see code for 5x --> +0.5

3. Question 1.2, 1.4 plotting script(s)

  * Code is a mix of 5x and 15x simulations --> +0.5

4. Question 1.2, 1.4 histograms with overlaid Poisson distributions (0.5 pts each)

  * good overall --> +1
  * if you would like a height matchup with the poisson distribution and the histogram, you could change the number of bins used in the histogram to match the number of unique coverage whole number values plotted/represented in the dataset

5. Question 1.3, 1.4 how much of genome not sequenced/comparison to Poisson expectations (0.5 pts each, 0.25 for answer and 0.25 for code)

  * good, but I only see code for 5x --> +0.75

6. Question 2 De novo assembly (0.5 pts each, 0.25 for answer and 0.25 for code)

  * number of contigs --> +0.5
  * total length of contigs --> +0.5

7. Question 2 De novo assembly cont (0.5 pts each, 0.25 for answer and 0.25 for code)

  * size of largest contig --> +0.5
  * contig n50 size --> +0.5

8. whole genome alignment (0.33 pts each, 0.33/2 for answer and 0.33/2 for code)

  * average identity --> +0.33
  * length of longest alignment --> +0.33
  * how many insertions and deletions in assembly --> number of insertions? 0.2475 (+ 0.33/4 for half answer + 0.33/2 for explanation/code)

9. decoding the insertion (0.5 pts each, 0.25 for answer and 0.25 for code)

  * position of insertion in assembly --> +0.5
  * length of novel insertion --> +0.5

10. decoding the insertion cont (0.5 pts each, 0.25 for answer and 0.25 for code)

  * DNA sequence of encoded message --> +0.5
  * secret message --> +0.5
