This is good work! A couple small points: 

2c. A python script would be great here, but awk is also a good way of manipulating this data. 

3c. You are correct that 7 has the most features. However, the question is asking about a way to figure out which state takes up the largest portion of the chromosome. It's quite possible that many of the state 7 genomic regions are relatively small – in this case, even with all the state 7 regions, it could occupy a small fraction of the total chromosome. No need to code it, but how would you find the state that represents the largest portion of chr21?

4c. This totally works, yes. Especially since you only have 5 superpopulations, so running the code 5 times is not too hard. If you had more, you could also cut out multiple fields and find unique combinations of those.

5d. Searching for ";AF=1" is a great way to avoid counting lines with things like "SAS_AF=1"

5e. You're right that there is only one recorded global allele frequency, but it is possible for "AF=1" to occur more than once, as in the case of a line with "...EUR_AF=1...EAS_AF=1". This is exactly what you control for very well in the previous section. 

Great work overall!

– Andrew
